test/avocado/timeoutlib.py

```diff
@@ -52,7 +52,7 @@ def __exit__(self, *args, **kwargs):
         pass
 
 class Retry(object):
-    def __init__(self, attempts = 1, timeout = None, exceptions = (Exception,), error = None, inverse = False, delay = None):
```

Comment:
timeoutlib.py has some tests below, including a comment "# And react only to a set of exceptions" which no longer makes sense (the "only").  Could you change this comment and maybe add a new testcase that verifies the new default behaviour?

Reply:
Fixed.